tests/unit/FileUploadPreview.spec.ts

```diff
@@ -35,7 +35,7 @@ function myArrayBuffer () {
   })
 }
 
-describe('FileUploadPreview component', () => {
+xdescribe('FileUploadPreview component', () => {
```

Comment:
There's a webpack error in this test due to the updated pdfjs-dist package. I'll keep trying to figure this out.

Reply:
Fixed now.